app/controllers/application_controller.rb

```diff
@@ -130,7 +130,10 @@ def skip_session_expiration
   end
 
   def set_locale
-    I18n.locale = params[:locale] || I18n.default_locale
+    I18n.locale =
+      http_accept_language.compatible_language_from(I18n.available_locales) ||
+      params[:locale] ||
+      I18n.default_locale
```

Comment:
ah ok! no problem. Then, let's strip the params[:locale] from this PR and push without my commit in there.